stable/rabbitmq/values-production.yaml

```diff
@@ -0,0 +1,170 @@
+## Bitnami RabbitMQ image version
+## ref: https://hub.docker.com/r/bitnami/rabbitmq/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/rabbitmq
+  tag: 3.7.4-r1
```

Comment:
can you update this tag to `3.7.4`? The tag already exists.

Reply:
ok, I didnt know if that was the case. will do.